### PR TITLE
fix cost update bug

### PR DESCRIPTION
**Pull request by andreram**
@JoshLuong please rigorously test

### CODE REVIEW

File: backend/src/routes/itineraries.ts [hunk 1]
```diff
@@ -44,8 +44,8 @@ router.get("/:id", async (req: any, res, _next) => {
   const { id } = req.params;
   const itinerary = await Itinerary.findOne({
     $or: [
-      { _id: id, user_id: req.session.userId },
```
Comment (JoshLuong) [inline backend/src/routes/itineraries.ts]: was this just for testing?

Comment (andreram) [reply backend/src/routes/itineraries.ts]: o yea, i'll revert that!

**Comment by JoshLuong** [comment]
does this PR include ur code clean up tasks?

**Comment by r-bassi** [comment]
Tested the cost editing and it seems good to me. Once done the code cleanup and resolve conflicts we can merge?

**Comment by andreram** [comment]
@JoshLuong It started as code cleanup and then I switched to fixing the bug when you mentioned it. Gonna leave the rest of the cleanup to another PR.

**Review by JoshLuong — APPROVED** [review]
LGTM

**Pull request merged**